bedrock/mozorg/templates/mozorg/contact/contact-base.html

```diff
@@ -6,12 +6,15 @@
 
 {% from "mozorg/contact/includes/macros.html" import map_image with context %}
```

Comment:
**suggestion:** the `map_image` macro (and it's code in the include) can be removed?